src/java.base/share/classes/java/time/MonthDay.java

```diff
@@ -748,10 +750,12 @@ public int hashCode() {
      */
     @Override
     public String toString() {
-        return new StringBuilder(10).append("--")
-            .append(month < 10 ? "0" : "").append(month)
-            .append(day < 10 ? "-0" : "-").append(day)
-            .toString();
+        StringBuilder buf = new StringBuilder(10);
```

Comment:
Why is this set to 10 instead of 7?

Reply:
I suggest not making this change, as it is irrelevant to this PR.